---
layout: default
---

<!DOCTYPE html>
<html>
<a href="https://genomerik.github.io/TranscriptoRIK">Home</a>
  
<head>
<title>Page Title</title>
</head>
<body>

<h1>Getting started with TAMA Collapse</h1>
<p>TAMA Collapse is a tool used primarily to remove redundant transcript models from long read transcript sequencing read mapping files.</p>

<p>
The input for TAMA Collapse is a BAM/SAM file generated by mapping long read transcript sequencing data onto a genome assembly. 
The BAM/SAM file must be sorted first which can be done using Samtools. 

You can find a demo SAM file in the TAMA reopository here:
<a href="https://github.com/GenomeRIK/tama/blob/master/test_files/gmap_test.sam">test.sam</a>
</p>


<p>
  You will also need the reference genome fasta file that was used for the read mappings. 
 
  You can find a demo Fasta file in the TAMA reopository here:
  <a href="https://github.com/GenomeRIK/tama/blob/master/test_files/test_genome.fa">test_genome.fa</a>
</p>

<p>
  To run TAMA Collapse with default parameters you can use this command: 
</p>

<p>
  <em>
    python tama_collapse.py -s test.sam -f test_genome.fa  
  </em>
</p>

<p>
  Note that in the default setting these parameters are used:
  -a 10
  -m 10
  -z 10
  -x capped
</p>
  
</body>
</html>
